blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,191 @@
+- **Node Image channel:** Keep node images updated with the latest security patches and OS improvements
+- **Planned maintenance windows:** Schedule upgrades during off-peak hours to minimize business impact
+
+For comprehensive guidance on all available upgrade strategies, see the [AKS cluster upgrade documentation](https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster). These production-tested upgrade mechanisms ensure smooth transitions between LTS versions while maintaining workload availability.
+
+## Looking Ahead
+
+AKS LTS represents our commitment to supporting enterprises at every stage of their Kubernetes journey. While LTS focuses on stability, we continue to innovate rapidly in standard AKS, ensuring you have access to the latest Kubernetes capabilities when you need them.
+
+Over the coming months, we'll be expanding AKS LTS capabilities based on your feedback, including:
+
+- **Istio support for LTS:** Bringing comprehensive LTS support for Istio service mesh to provide enterprise-grade stability for your microservices architecture
+
+**Enhanced AKS Upgrade Capabilities (coming soon for both standard AKS and LTS Support Plan):**
+- **Agent pool Blue-Green upgrades:** Node pool-level blue-green upgrade strategy that enables workload validation batch by batch, with the ability to rollback newly created green nodes within a configurable soak period
+- **Component Version API:** A dedicated API to surface breaking changes in AKS components, including AKS-managed add-ons and OS components, helping customers understand compatibility impacts before upgrading to the next AKS LTS version
+- **Enhanced PDB management:** Simplified Pod Disruption Budget creation and management capabilities to streamline PDB setup and reduce upgrade complexity
+
+## Real-World Customer Scenarios
+
+**Financial Services:** A major bank running regulatory compliance workloads told us: *"We need Kubernetes for modern app development, but our compliance team requires extended stability guarantees. AKS LTS gives us both innovation and the predictability our auditors demand with 24 months of support."*
+
+**Healthcare:** A healthcare provider managing patient data systems shared: *"Frequent cluster upgrades mean extensive testing and validation cycles. With AKS LTS, we can focus on improving patient outcomes instead of constant infrastructure maintenance."*
+
+**Manufacturing:** An IoT platform managing factory operations explained: *"Our edge clusters run critical production line controls. Unexpected upgrades could halt manufacturing. AKS LTS gives us the stability to keep factories running while still benefiting from modern Kubernetes capabilities."*
+
+## Azure Linux Support for AKS LTS
+
+**Azure Linux now supports AKS Long Term Support**, starting with Kubernetes v1.29. This expands your options for building stable, enterprise-grade Kubernetes infrastructure on Microsoft's own Linux distribution.
```

Comment:
we should also capture baseline of before - were all other OS options of AKS already available with LTS and Azure Linux is the last piece of the puzzle now?

Reply:
we can also use the official terminology as stated in Azure Linux docs - "Azure Linux Container Host for AKS" along with a link to https://learn.microsoft.com/en-us/azure/azure-linux/intro-azure-linux. speaking of which the key benefits listed under that doc (https://learn.microsoft.com/en-us/azure/azure-linux/intro-azure-linux#azure-linux-container-host-key-benefits) seem to be hitting the core terminologies (secure supply chain, secure by default...). we can probably borrow from there to stay consistent